drake/systems/plants/collision/BulletModel.cpp

```diff
@@ -299,6 +299,40 @@ namespace DrakeCollision
     return c.getResults();
   }
 
+  bool BulletModel::collidingPointsCheckOnly(const vector<Vector3d>& points,
+                                             double collision_threshold)
+  {
+    // Create sphere geometry
+    btSphereShape bt_shape(collision_threshold);
+
+    // Create Bullet collision object
+    btCollisionObject bt_obj;
+    btCollisionObject* bt_obj_ptr = static_cast<btCollisionObject*>(&bt_obj);
```

Comment:
@avalenzu yes, it doesn't seem to be necessary so I removed it. Thanks for reviewing the code.